SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -50,6 +50,18 @@ bool USpatialStatics::IsSpatialNetworkingEnabled()
 	return GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking();
 }
 
+bool USpatialStatics::IsHandoverEnabled(const UWorld* World)
```

Comment:
Can we make this take a WorldContextObject similar to other SpatialStatics methods? Also should be just be returning true if the World is null? I'd think we should warn? Is there a valid case where we'd want to call this before the World was initialized and would want the return to just be true?

Reply:
Also added a warning and the checks which are present in the other WorldContext functions! Not sure about calling this before the World is initialized, I guess I changed a dependency here moving the setting into the multi-worker settings. Does moving this setting make sense? Is it better as a project-wide setting? I guess there is potential to have offloaded/zoned maps in the same project but maybe this is so rare it doesn't matter?